examples/sph/run_sod.py

```
@@ -26,23 +38,33 @@
 u_g = P_g / ((gamma - 1) * rho_g)
 u_d = P_d / ((gamma - 1) * rho_d)
 
-resol = 128
+resol = int(os.environ.get("RESOL", 128))
```

Ruff (0.15.21)

[warning] Invalid type for environment variable default; expected `str` or `None`

(PLW1508)